Mathlib/Analysis/Complex/Angle.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Affine
+import Mathlib.Analysis.SpecialFunctions.Complex.Arg
+
+/-!
+# Angle between complex numbers
+
+This file relates the Euclidean geometric notion of angle between complex numbers to the argument of
+their quotient.
+
+## Tags
+
+arc-length, arc-distance
+-/
+
+open InnerProductGeometry
+open scoped Real
+
+namespace Complex
+variable {a x y : ℂ}
+
+lemma angle_eq_abs_arg (hx : x ≠ 0) (hy : y ≠ 0) : angle x y = |(x / y).arg| := by
```

Comment:
I think it's worth adding a docstring reminding the reader why this is not true when `x` or `y` is 0.